### ISSUE TITLE
[docker] sh: git: not found on php artisan calls

### ISSUE THREAD

**Issue by tlvince** [monicahq/monica#947]
On the latest Docker image (v1.8.0), any calls to `php artisan` return `sh: git: not found`, but do seem to exit successfully. For example, on startup:

```
monicahq_1  | /var/www/monica/scripts/docker/waitfordb.sh
monicahq_1  | Connecting to mysql:3306
monicahq_1  | Waiting for database to settle ...
monicahq_1  | /var/www/monica/scripts/docker/test-server.sh
monicahq_1  | APP_KEY already set
monicahq_1  | sh: git: not found
monicahq_1  | Nothing to migrate.
monicahq_1  | sh: git: not found
monicahq_1  | The "public/storage" directory already exists.
monicahq_1  | sh: git: not found
monicahq_1  | sh: git: not found
```

Or in shell:

```
❯ docker-compose run monicahq shell
/var/www/monica # php artisan send:reminders
sh: git: not found
```

Probably `git` needs adding to the Docker image?

**Comment by asbiin** [comment]
No, git is not needed.
The message is related to #872, but it's not an error.
Adding git will not be usefull as the .git directory is no longer part of docker image.

**Comment by tlvince** [comment]
Closed by #950.

**Comment by asbiin** [comment]
version 1.8.1 fixed this
